enterprise/utils.py

```diff
@@ -438,25 +438,14 @@ def get_notification_subject_line(course_name, template_configuration=None):
         return stock_subject_template.format(course_name=course_name)
 
 
-def send_email_notification_message(user, enrolled_in, enterprise_customer, email_connection=None):
```

Comment:
again not convinced we couldn't re-use this.

Reply:
yeah I guess I can still reuse this method with an optional bulk enroll flag passed to it? that seems doable, but it will still need me to add branching logic since for bulk enroll I was going to not use the fallback text/html files...but other than that it's largely reusable